x-pack/plugins/enterprise_search/public/applications/shared/indexing_status/indexing_status_logic.test.ts

```diff
@@ -24,7 +26,8 @@ describe('IndexingStatusLogic', () => {
 
   beforeEach(() => {
     jest.clearAllMocks();
-    mount();
+    resetContext({});
```

Comment:
You shouldn't need this if using the LogicMounter, it resets context automatically!

```suggestion
```